app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -922,19 +922,27 @@ struct DeviceDelegateOculusVR::State {
           const bool yTouched = (state[i].controllerState.Touches & ovrTouch_Y) != 0;
           const bool menuPressed = (state[i].controllerState.Buttons & ovrButton_Enter) != 0;
 
-          controller->SetButtonState(i, ControllerDelegate::BUTTON_APP, -1, menuPressed, menuPressed);
+
           controller->SetButtonState(i, ControllerDelegate::BUTTON_X, 3, xPressed, xTouched);
           controller->SetButtonState(i, ControllerDelegate::BUTTON_Y, 4, yPressed, yTouched);
+
+          if (renderMode != device::RenderMode::Immersive) {
+            controller->SetButtonState(i, ControllerDelegate::BUTTON_APP, -1, yPressed, yTouched);
+          } else {
+            controller->SetButtonState(i, ControllerDelegate::BUTTON_APP, -1, menuPressed, menuPressed);
```

Comment:
could we have the Menu button also recognised as Exit in Fullscreen Movie Mode?

Reply:
FWIW, Oculus Browser uses Y and B to Exit Fullscreen and its Movie Mode?

Reply:
tested both again. this behaviour is same as Oculus Browser's, so you can ignore this comment. but see comment below about toggling the Media Controls.